### ISSUE TITLE
Documents should include a "skip to Main" link

### ISSUE THREAD

**Issue by ozross** [TeXUsersGroup/TeX-VPAT#6]
WCAG Level A, Success Criterion 2.4.1 is about having a "Skip to main content link".

Definition:
Provide a keyboard-supported means of bypassing blocks of content, such as navigation menus and page headers, to get to the main content of the page.

[texlive-vpat-titlepage.pdf](https://github.com/TeXUsersGroup/TeX-VPAT/files/11144274/texlive-vpat-titlepage.pdf)
The attached PDF has such a link, as shown here:

<img width="660" alt="Skip-to-Main" src="https://user-images.githubusercontent.com/11621433/229676993-2d6d4f53-315f-4a54-b9f5-9f9e35c280b6.png">

This is quite easily done in conjunction with a logo, using a (pdfTeX) fake-space as the anchor text for the link, with target being the 1st section of main content. The destination name should be the same as the Bookmark name for that (sub-)section.
